Ryujinx.Ava/Helper/LoggerAdapter.cs

```diff
@@ -0,0 +1,163 @@
+using Avalonia.Utilities;
+using System;
+using System.Text;
+
+namespace Ryujinx.Ava.UI.Helper
+{
+    using AvaLogger = Avalonia.Logging.Logger;
+    using AvaLogLevel = Avalonia.Logging.LogEventLevel;
+    using RyuLogger = Ryujinx.Common.Logging.Logger;
+    using RyuLogClass = Ryujinx.Common.Logging.LogClass;
+
+    public class LoggerAdapter : Avalonia.Logging.ILogSink
+    {
+        public static void Register()
+        {
+            AvaLogger.Sink = new LoggerAdapter();
+        }
+
+        private static RyuLogger.Log? GetLog(AvaLogLevel level) => level switch {
```

Comment:
```suggestion
        private static RyuLogger.Log? GetLog(AvaLogLevel level)
        {
            return level switch
            {
```
With the appropriate indentation changes.